packages/terra-application-docs/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@cerner/terra-application-docs",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Contains documentation for packages in the terra-application monorepo",
+  "main": "index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-application.git",
+    "directory": "packages/terra-application-docs"
+  },
+  "keywords": [
+    "eslint",
+    "eslint-config",
```

Comment:
These are probably copy/paste. Do we actually want to keep these as keywords? Other doc packages have the same keywords so maybe it's ok.

Reply:
lol, yeah these keywords are probably all over the place in our doc packages